pitr: update limitation of first time restore (#16043)
pingcap/docs: br/br-pitr-manual.md

The example output only shows the common parameters. These parameters are described as follows:

> - `--full-backup-storage`: the storage address for the snapshot (full) backup. To use PITR, specify this parameter and choose the latest snapshot backup before the restore timestamp. To restore only log backup data, you can omit this parameter. Notice that when initializing the recovery cluster for the first time, must specify a backup for snapshot. Currently, BR supports Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [URI Formats of External Storage Services](/external-storage-uri.md).

Reviewer: ```suggestion
- `--full-backup-storage`: the storage address for the snapshot (full) backup. To use PITR, specify this parameter and choose the latest snapshot backup before the restore timestamp. To restore only log backup data, you can omit this parameter. Note that when initializing the recovery cluster for the first time, you must specify a snapshot backup. Currently, BR supports Amazon S3, GCS, or Azure Blob Storage as the storage for log backup. For details, see [URI Formats of External Storage Services](/external-storage-uri.md).
```

> **Note:**
>

> > - When you restore the cluster for the first time, it is necessary to specify the full snapshot data, otherwise some data in new created table may be incorrect due to rewriting Table ID rules.

Reviewer: ```suggestion
> - When you restore the cluster for the first time, you must specify the full snapshot data. Otherwise, some data in newly created table might be incorrect due to rewriting Table ID rules.
```
